libraries/AP_HAL_ChibiOS/Util.h

```diff
@@ -39,6 +39,11 @@ class ChibiOS::Util : public AP_HAL::Util {
     bool run_debug_shell(AP_HAL::BetterStream *stream) override { return false; }
     uint32_t available_memory() override;
 
+    // get path to custom defaults file for AP_Param
+    const char* get_custom_defaults_file() const override {
+        return "@ROMFS/defaults.parm";
```

Comment:
I'd like to see this logged to bin logs (added to the list)

Reply:
Done